heroes-sidebar-master/freeRotation.py

```diff
@@ -13,43 +15,25 @@ def __init__(self):
 		self.freeRotation = None;
 
 	def scrapeRotation(self):
-		try:
-			forum_page = requests.get('http://us.battle.net/heroes/en/forum/topic/17936383460')
-		except request.exceptions.ConnectionError:
-			data = json.loads(urllib.urlopen("http://ip.jsontest.com/").read())
-			email = email_me()
-			message = 'Heroes Bot Didnt Update\n' + 'The Bot didn\'t update because it couldn\'t get ROTATION from Blizzard Forums. Check it out.\nThe IP is ' + str(data["ip"])
-			email.populate_email('Heroes Bot Fail', message)
-			email.send_email()
-			sys.exit("Couldn't get ROTATION from Blizzard Forums")
+	    # We prefer the PhantomJS driver to avoid opening any GUI windows.
+	    browser = webdriver.PhantomJS()
```

Comment:
Eventually, I'd like to pull this portion of code that fetches the data from the website into a separate module. For now, it can live here.